.github/workflows/ci.yaml

```diff
@@ -87,6 +89,8 @@ jobs:
     needs: release
     runs-on: ubuntu-arm64-latest
     steps:
+      - name: Prune docker images
+        run: docker system prune -af
```

Comment:
I'll remove this after it runs once. I think it just needs to do a quick cleanup and then in general, it should be enough to run it at the end!

Reply:
oh makes sense!